features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/TimelineItemRow.kt

```diff
@@ -120,7 +130,22 @@ internal fun TimelineItemRow(
                         )
                     }
                     else -> {
+                        val a11yVoiceMessage = stringResource(CommonStrings.a11y_voice_message)
                         TimelineItemEventRow(
+                            modifier = Modifier
+                                .semantics(mergeDescendants = true) {
+                                    if (timelineItem.content is TimelineItemVoiceContent) {
+                                        val voiceMessageText = String.format(a11yVoiceMessage, timelineItem.content.duration.toString(DurationUnit.MINUTES))
+                                        contentDescription = "${timelineItem.safeSenderName}, $voiceMessageText"
```

Comment:
I cannot make the screen reader to read this string, because a `else` is missing below.
`contentDescription` is overridden by `timelineItem.safeSenderName`